src/renderer/helpers/poolHelper.test.ts

```diff
@@ -1,21 +1,21 @@
-import { BNBChain } from '@xchainjs/xchain-binance'
 import { BTCChain } from '@xchainjs/xchain-bitcoin'
 import { BCHChain } from '@xchainjs/xchain-bitcoincash'
+import { BSCChain } from '@xchainjs/xchain-bsc'
 import { Balance, Network } from '@xchainjs/xchain-client'
 import { GAIAChain } from '@xchainjs/xchain-cosmos'
 import { DOGEChain } from '@xchainjs/xchain-doge'
 import { ETHChain } from '@xchainjs/xchain-ethereum'
 import { LTCChain } from '@xchainjs/xchain-litecoin'
 import { PoolDetail } from '@xchainjs/xchain-midgard'
+import { AssetBSC } from '@xchainjs/xchain-thorchain-query'
```

Comment:
don't import asset from here.... it lives in '@xchainjs/xchain-bsc'

Reply:
correct. 
They actually should be imported from ../../shared/utils/asset